helm/superset/templates/_helpers.tpl

```diff
@@ -90,14 +90,22 @@ WTF_CSRF_EXEMPT_LIST = []
 # A CSRF token that expires in 1 year
 WTF_CSRF_TIME_LIMIT = 60 * 60 * 24 * 365
 class CeleryConfig(object):
-  BROKER_URL = f"redis://{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
   CELERY_IMPORTS = ('superset.sql_lab', )
-  CELERY_RESULT_BACKEND = f"redis://{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
   CELERY_ANNOTATIONS = {'tasks.add': {'rate_limit': '10/s'}}
+{{- if .Values.supersetNode.connections.redis_password }}
```

Comment:
I think the check of `auth.enabled` is needed here. Otherwise, the redis without password is connected with an incorrect url because the default value in `value.yaml` is set.